src/test/resources/general.conf

```diff
@@ -41,6 +41,7 @@ docker {
 }
 
 jdbc-journal {
+  logicalDelete = ${akka-persistence-jdbc.logicalDeletion.enable}
```

Comment:
I don't think it is actually needed to include this, since this is the same as in the reference.conf (the same holds for `includeLogicallyDeleted` below)

Reply:
Actually we need it, but this is needed only because of how the config is being loaded in the tests. The var replacement was just not working as expected, so I have to re-add it here. 

Basically, the var replacement happens too early. First, it loads `reference.conf` and it resolves the var to `true`, then it loads the test config and merges with the resolved `reference`, but at that point, there is nothing else to replace.

I tested with a real application and it works. We can configure `akka-persistence-jdbc.logicalDeletion.enable` and it propagates to the two others as expected.